service-api/module/Application/src/Services/Experian/FraudApi/ExperianCrosscoreFraudApiService.php

```diff
@@ -0,0 +1,172 @@
+{
+    public function __construct(
+        private readonly Client $client,
+        private readonly ExperianCrosscoreAuthApiService $experianCrosscoreAuthApiService,
+        private readonly array $config
+    ) {
+    }
+
+    /**
+     * @throws GuzzleException
+     * @throws ExperianCrosscoreAuthApiException
+     */
+    public function makeHeaders(): array
+    {
+        return [
+            'Content-Type' => 'application/json',
+            'Authorization' => $this->experianCrosscoreAuthApiService->retrieveCachedTokenResponse(),
+            'X-User-Domain' => $this->config['domain']
+        ];
+    }
+
+    /**
+     * @throws GuzzleException
+     * @throws ExperianCrosscoreFraudApiException
+     */
+    public function getFraudScore(
+        ExperianCrosscoreFraudRequestDTO $experianCrosscoreFraudRequestDTO
+    ): array {
+        $response = $this->makeRequest($experianCrosscoreFraudRequestDTO);
+        return $response->toArray();
+    }
+
+    /**
+     * @throws GuzzleException
+     * @throws ExperianCrosscoreFraudApiException
+     * @psalm-suppress InvalidReturnType
+     */
+    public function makeRequest(
+        ExperianCrosscoreFraudRequestDTO $experianCrosscoreFraudRequestDTO
+    ): ExperianCrosscoreFraudResponseDTO {
+        try {
+            $postBody = $this->constructRequestBody($experianCrosscoreFraudRequestDTO);
+
+            $response = $this->client->request(
+                'POST',
+                '/3',
+                [
+                    'headers' => $this->makeHeaders(),
+                    'json' => json_encode($postBody)
+                ]
+            );
+
+            $responseArray = json_decode($response->getBody()->getContents(), true);
+
+            return new ExperianCrosscoreFraudResponseDTO(
+                $responseArray
+            );
+        } catch (ClientException $clientException) {
+            if ($clientException->getResponse()->getStatusCode() == Response::STATUS_CODE_401) {
+                $this->experianCrosscoreAuthApiService->authenticate();
+                $this->makeRequest($experianCrosscoreFraudRequestDTO);
+            }
+        } catch (\Exception $exception) {
+            throw new ExperianCrosscoreFraudApiException($exception->getMessage());
+        }
+    }
+
+    public function constructRequestBody(
+        ExperianCrosscoreFraudRequestDTO $experianCrosscoreFraudRequestDTO
+    ): array {
+        $requestUuid = Uuid::uuid4()->toString();
+        $personId = $this->makePersonId($experianCrosscoreFraudRequestDTO);
+        $addressDTO = $experianCrosscoreFraudRequestDTO->address();
+
+        $body = [
+            'header' => [
+                "tenantId" => $this->config['tenantId'],
+                "requestType" => "FraudScore",
+                "clientReferenceId" => "$requestUuid-FraudScore-continue",
+                "expRequestId" => $requestUuid,
+                "messageTime" => date("Y-m-d\TH:i:s.000\Z", time()),
```

Comment:
`time()` is the default second argument of `date()`
```suggestion
                "messageTime" => date("Y-m-d\TH:i:s.000\Z"),
```